sqlmesh/core/renderer.py

```diff
@@ -211,12 +212,16 @@ def _resolve_table(table: str | exp.Table) -> str:
                     expressions = [e for e in parse(rendered_expression, read=self._dialect) if e]
 
                     if not expressions:
-                        raise ConfigError(f"Failed to parse an expression:\n{self._expression}")
+                        raise ConfigError(
+                            f"{traceback.format_exc()}"
```

Comment:
Why do we include the stacktrace here? This is just parsing right?

Reply:
If you're doing this for jinja's sake then I don't think it makes sense. As far as I remember jinja's stacktrace is pretty useless